### ISSUE TITLE
Intermittent issue with packages install

### ISSUE THREAD

**Issue by loshkovskyi** [CiscoCloud/mantl#1189]
Intermittently Mantl deployment fails with the messages like:
```
TASK: [common | install distributive] ***************************************** 
failed: [mantl-andrii-control-03] => {"changed": false, "failed": true, "rc": 1}
msg: Package at https://bintray.com/artifact/download/asteris/mantl-rpm/distributive-0.2.4-1.x86_64.rpm could not be installed
failed: [mantl-andrii-control-02] => {"changed": false, "failed": true, "rc": 1}
msg: Package at https://bintray.com/artifact/download/asteris/mantl-rpm/distributive-0.2.4-1.x86_64.rpm could not be installed
changed: [mantl-andrii-worker-003]
changed: [mantl-andrii-control-01]
changed: [mantl-andrii-worker-002]
changed: [mantl-andrii-edge-01]
changed: [mantl-andrii-edge-02]
changed: [mantl-andrii-worker-001]

TASK: [common | create consul configuration directory] ************************ 
changed: [mantl-andrii-control-01]
changed: [mantl-andrii-worker-001]
changed: [mantl-andrii-edge-02]
changed: [mantl-andrii-edge-01]
changed: [mantl-andrii-worker-002]
changed: [mantl-andrii-worker-003]
```
This issue is not related to one specific package or a host but rather to all packages installed via the `yum` Ansible module.
Sometimes, package is not installed on all hosts and sometimes only on several of them.
Pay attention to the task following the failed one. Now, Ansible determines that there are only 6 hosts left instead of 8, so all further tasks will be applied for the remained 6 hosts.

I'm able to reproduce this problem in an Amsterdam-based OpenStack cloud. For example, in a Texas cloud this issue is not present.

It could be either a cloud network problem or a Bintray CDN problem, or something else. Although the hosts are accessible all of the time - they are not getting down or frozen.

I think we should handle such problems. A possible work around, which I've successfully tested:

In the `common` role add these yum repositories:
```
#bintraybintray-ciscocloud-rpm - packages by ciscocloud from Bintray
[bintraybintray-ciscocloud-rpm]
name=bintray-ciscocloud-rpm
baseurl=https://dl.bintray.com/ciscocloud/rpm
gpgcheck=0
enabled=1

#bintraybintray-asteris-mantl-rpm - packages by asteris from Bintray
[bintraybintray-asteris-mantl-rpm]
name=bintray-asteris-mantl-rpm
baseurl=https://dl.bintray.com/asteris/mantl-rpm
gpgcheck=0
enabled=1
```
Then for all packages:
```
   yum:
-    name: https://bintray.com/artifact/download/asteris/mantl-rpm/consul-template-0.12.2-1.x86_64.rpm
+    name: consul-template-0.12.2-1
```
So I've tested and this change works well for all environments.

There are some minor issues though. Why do some repos are in CiscoCloud and some in Asteris? There should be one company, in my opinion. Moreover, there are some packages that are installed from completely 3-rd party repos.

Also, looks like metadata is broken in some packages. I'm able to do:
```
yum install consul-template-0.12.2-1
```
but not:
```
yum install distributive-0.2.4-1
```

**Comment by stevendborrelli** [comment]
Thanks! This sorely needed.

**Comment by stevendborrelli** [comment]
Also see #1188. We'd like to get a single repo.

**Comment by ryane** [comment]
this is mostly addressed by #1204 and #1193. #1188 captures the singe repo issue. thanks!
